### PR TITLE
Improve code style: move hardcoded values to constants

### PR DESCRIPTION
**Pull request by dkhmelenko**

### CODE REVIEW

**Comment by ShivamDev31** [comment]
Shouldn't ShadowActivity extends AppCompatActivity for backward compatibility. I know it won't effect the library user. Just keen to know.

**Comment by dkhmelenko** [comment]
@ShivamDev31 AppCompatActivity requires appcompat-v7 dependency. I don't see any reason to have it and prefer to keep the library more lightweit

**Comment by boyw165** [comment]
I don't know if lightweight is that important over the compatibility. Please check out the [issue](https://github.com/tbruyelle/RxPermissions/issues/59) thread. I personally think it's necessary to have this library to support API of <21.

File: lib/src/main/java/com/tbruyelle/rxpermissions/ShadowActivity.java [hunk 1]
```diff
@@ -7,7 +7,11 @@
 import android.os.Bundle;
 
 @TargetApi(Build.VERSION_CODES.M)
-public class ShadowActivity extends Activity {
+class ShadowActivity extends Activity {
```
Comment (tbruyelle) [inline lib/src/main/java/com/tbruyelle/rxpermissions/ShadowActivity.java]: Why did you change the class scope ?

**Comment by tbruyelle** [comment]
Although your changes refer to good old java practrices, I don't want to merge them because I think they do not promote code readability.
